source/server/spongineer/incompatible.rst

```diff
@@ -84,11 +84,13 @@ Just Enough IDs (JEID)
 - Problem: Crash on startup (mixin conflict).
 - Solution: Update to version 1.0.3-54 or newer.
 
-LagGoggles
-~~~~~~~~~~
-- Versions: Older than 1.12.2-5.3-113
+LagGoggles/TickCentral
+~~~~~~~~~~~~~~~~~~~~~~
+- Versions: All
 - Problem: Crash on startup (mixin conflict).
-- Solution: Update to version 1.12.2-5.3-113 and install *TickCentral* core-mod.
+- Solution: Due to the very invasive nature of these mods, SpongeForge cannot run properly alongside them. 
```

Comment:
To be honest, we don't really have to dig into the nuts-and-bolts of why it goes wonky. I think this could use some diplomatic rephrasing, too.
"We suggest only using this mod when needed to debug issues. When used alongside SpongeForge, compatibility and game stability cannot be assured. See here for more info [Link]"

I'd like a more clear-cut solution, like "do not use with SpongeForge", but at least it gets around keeping a table of supposedly compatible versions.

Reply:
I'll just make it a "do not use" like mystcraft, because there isn't a real way to run both at once since they crash.